### PR TITLE
university student project edits

### PR DESCRIPTION
**Pull request by compSciKai**
Hello! I am a university student taking a technical writing course. For one project we need to make suggested edits to repos on github. 
Please use the following suggestions at your discretion :)

- removed instances of hedging, and contractions to increase formal language
- reduced pronoun use to reduce ambiguity
- minor edits to increase clarity

**Proposed changes**:
- close https://github.com/RasaHQ/rasa/issues/4836

**Status (please check what you already did)**:
- [ ] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,12 +26,12 @@ or voice assistants as:
 - Alexa Skills
 - Google Home Actions
 
-Rasa's primary purpose is to help you build contextual, layered
+Rasa's primary purpose is to help build contextual, and layered
```
Comment (erohmensing) [inline README.md]:
```suggestion
Rasa's primary purpose is to help build contextual, layered
```

File: README.md [hunk 2]
```diff
@@ -25,12 +26,12 @@ or voice assistants as:
...
 conversations with lots of back-and-forth. To have a real conversation,
-you need to have some memory and build on things that were said earlier.
+you need to use your memory and build on things said previously.
```
Comment (erohmensing) [inline README.md]:
```suggestion
your bot needs to have some memory and build on things said previously.
```

File: README.md [hunk 3]
```diff
@@ -25,12 +26,12 @@ or voice assistants as:
...
 Rasa lets you do that in a scalable way.
 
-There's a lot more background information in this
+There is more background information here:
```
Comment (erohmensing) [inline README.md]:
```suggestion
Find more background information in this
```

File: README.md [hunk 4]
```diff
@@ -59,8 +60,7 @@ There's a lot more background information in this
 ## Where to get help
 
 There is extensive documentation in the [Rasa Docs](https://rasa.com/docs/rasa).
-Make sure to select the correct version so you are looking at
-the docs for the version you installed.
+Make sure to select the correct version to use the correct corresponding documentation.
```
Comment (erohmensing) [inline README.md]:
```suggestion
Make sure to select the version you installed to see the correct corresponding documentation.
```

File: README.md [hunk 5]
```diff
@@ -72,7 +72,7 @@ questions.
 
 ### How to contribute
 We are very happy to receive and merge your contributions. You can
-find more information about how to contribute to Rasa (in lots of
+find more information about contributing to Rasa (in lots of
```
Comment (erohmensing) [inline README.md]:
```suggestion
find more information about how to contribute to Rasa (in lots of
```

**Comment by compSciKai** [comment]
Thank you for your time!

**Comment by erohmensing** [comment]
One last thing, can you please sign the CLA? https://github.com/RasaHQ/rasa/pull/4837#issuecomment-558456956

**Comment by tmbo** [comment]
@compSciKai do you mind signing the contributor agreement? https://github.com/RasaHQ/rasa/pull/4837#issuecomment-558456956

**Comment by tmbo** [comment]
I’ll close this for now - let us know if you still want to move this forward

**Comment by compSciKai** [comment]
Hi @tmbo 
I have tried going through the portal to sign in the correct channels without success. Any suggestions on how to sign successfully?

**Comment by erohmensing** [comment]
@compSciKai maybe its not working because the PR is closed? 🤔 What do you mean by "without success" -- whats happening when you try?

**Comment by tmbo** [comment]
Not sure what you mean with "channels", you only need to click on the button and fill out the form while being logged in to github.

I'll close this, let me know once you signed it.

**Comment by compSciKai** [comment]
I clicked sign, it tells me that I signed it, and I come back here and it says I haven't signed it. 
You can close this request if you would like. Must be a bug.

**Pull request closed without merging**